### ISSUE TITLE
feature to disable on tlp

### ISSUE THREAD

**Issue by blazessdd** [konkor/cpufreq#116]
what are the feature to disable to get full cpu control on cpu freq?
because if you change the power profile the system goes on overlad due to the concurrency of tlp and cpufreq to set the same

**Comment by midjean** [comment]
Hello,
I would also like to know as it seems that they are conflicting too on my laptop.
My thanks,
Jean

**Comment by konkor** [comment]
I don't think that the application should monitor/enable/disable others power management solutions. You shouldn't use the **On Battery** event in the preferences window. Otherwise, you should disable cpu power management in TLP settings.

**Comment by konkor** [comment]
Maybe do you mean to disable applying the power profiles/settings **On Battery** system event?

**Comment by blazessdd** [comment]
I'm asking about what config conflict with applying different profile to CPU because it makes the system loading skyrocket and makes the computer slow

**Comment by konkor** [comment]
@blazessdd Don't get me wrong. The application has designed to manage CPU power profiles in first place. What it has to do exactly? How do you see it? Maybe I can't understand something.

**Comment by blazessdd** [comment]
If tlp is enabled and I change the power profile the application creates a dump in the home directory and the system becames unresponsive because there is a concurrency in setting the CPU power profile between cpufreq and tlp so I was asking what tlp line to change in order to manage the CPU only in cpufreq

**Comment by konkor** [comment]
1. The application doesn't create a dump any more. It was so on some version gjs. It fixed everywhere on gjs and extension side. BTW it was GJS profiling dump. It was misunderstanding of the command line option **--profile ID**
2. The default value for **On Battery** event is **do nothing** now. There was a concurrency in beta versions (before publishing it on _extensions.gnome.org_) when it tried to apply **Battery** profile by default.
3. It's like you have to delete/comment cpu scaling options in `/etc/defaults/tlp` [TLP configuration](https://linrunner.de/en/tlp/docs/tlp-configuration.html#scaling) or just use some GUI for it like [TLP UI](https://github.com/d4nj1/TLPUI) etc.
